src/test/java/hw3/ex2/HomeAndElementsFluentTest.java

```diff
@@ -0,0 +1,48 @@
+package hw3.ex2;
+
+import hw3.AbstractTestBase;
+import hw3.ex1.HomePageFluent;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class HomeAndElementsFluentTest extends AbstractTestBase {
+    @Test
+    public void logInGoToElementsTest() {
+        ElementsPageFluent elementsPage = new ElementsPageFluent(driver);
+        HomePageFluent homePage = new HomePageFluent(driver);
+        driver.get(credentialsHelper.getProperty("url"));
+        assertEquals(driver.getTitle(), "Home Page");
+
+        //logInAndCheckUserName
+        homePage.openLogInForm()
+                .enterUserName(credentialsHelper.getProperty("login"))
+                .enterPassword(credentialsHelper.getProperty("password"))
+                .clickLoginBtn();
+
+        assertEquals(homePage.getLoggedInUserName(), "ROMAN IOVLEV");
+
+        //getToElementsPage
+        homePage.clickOnTheServiceTab()
+                .clickOnTheDifferentElements()
+                .clickWaterCheckBox()//Select checkboxes Water, Wind
+                .clickWindCheckBox()
+                .clickSelenRadio()//Select radio selen
+                .clickColorsDropDown()
+                .clickYellow();//Select in dropdown Yellow
```

Comment:
It will be better send names of the checkbox, radiobutton and select as parameter to the page method